cmd/wallet-web/src/pages/CredentialDetailsPage.vue

```diff
@@ -190,18 +190,24 @@ export default {
       const user = profile.user;
       this.credentialManager = new CredentialManager({ agent: this.getAgentInstance(), user });
       try {
-        const { id, name, issuanceDate, resolved } =
+        let { id, name, issuanceDate, resolved } =
           await this.credentialManager.getCredentialMetadata(
             this.token,
             decode(this.$route.params.id)
           );
+        if (!name) name = resolved[0].title;
         this.credential = { id, issuanceDate, name, ...resolved[0] };
```

Comment:
Maybe we could do this instead:
```suggestion
        this.credential = { id, issuanceDate, name || resolved[0].title, ...resolved[0] };
```